templates/next-app-dir/app/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { Styles } from './styles';
+import { reader } from './reader';
+
+export default async function Homepage() {
+  const posts = await reader.collections.posts.all();
+
+  return (
+    <div>
+      <h1>Keystatic ⚡️</h1>
+      <p>This homepage shows how to load a collection from the reader API.</p>
+      <p>
+        <a href="/keystatic">Click here to visit the Admin UI</a>, or the link
+        below to view a post in the collection.
+      </p>
+      <h2>Posts</h2>
+      <ul>
+        {posts.map(post => (
+          <li>
```

Comment:
Should we put a `key` on here? i.e. `key={post.slug}`?

Reply:
💯